app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] Kindly make sure that the design matches the given wireframe.

![image](https://user-images.githubusercontent.com/62290545/186688686-ea0dafb0-ee3e-470e-a90e-c6ff3ea8d9a4.png)

Reply:
@ShadyShawkat This is the signup page and not login page.

The sign up page is not part of the wire frame but part of the requirement as the app is expected to register new users.
Following the wireframe exactly means i wont be able to register new user

Reply:
Hey @Adanzeakonobi, 
You're correct there's no exact wireframe for the signup page. I'll make this change optional. However, you can design all of the forms to match the form on the login page.